app/helpers/blazer/base_helper.rb

```diff
@@ -37,6 +37,10 @@ def blazer_maps?
       ENV["MAPBOX_ACCESS_TOKEN"].present?
     end
 
+    def blazer_image?(key, value)
+      key.to_s.end_with?("image") && value.is_a?(String) && %w[png jpg jpeg].include?(value.split(".").last)
```

Comment:
Also, would be good to look at extension between `.` and `?` so that urls like `http://.../image.gif?w=100&h=100` work